File: app/src/at-myhuynh/java/asiantech/internship/summer/JavaTrainning/src/ex03/TotalOfNumber.java
package ex03;

import common.Common;

public class TotalOfNumber {

	public static void main(String[] args) {
		int number = Common.input("Nhập vào số cần tính: ");
		int total = Common.totalOfNumber(number);

		System.out.println(number + " => " + total);
	}
}

File: app/src/at-myhuynh/java/asiantech/internship/summer/kotlin/ex03/TotalOfNumber.kt
package asiantech.internship.summer.kotlin.ex03

import asiantech.internship.summer.kotlin.common.Common
import asiantech.internship.summer.kotlin.common.totalOfNumber

object TotalOfNumber {

    @JvmStatic
    fun main(args: Array<String>) {
        val number = Common.input("Nhập vào số cần tính: ")
        val total = number.totalOfNumber()

        println("$number => $total")
    }
}